src/openMVG_Samples/features_repeatability/CMakeLists.txt

```diff
@@ -1,12 +1,17 @@
-
-ADD_EXECUTABLE(openMVG_sample_main_repeatability_dataset main_repeatability_dataset.cpp)
-TARGET_LINK_LIBRARIES(openMVG_sample_main_repeatability_dataset
+add_executable(openMVG_sample_main_repeatability_dataset main_repeatability_dataset.cpp)
+target_link_libraries(openMVG_sample_main_repeatability_dataset
   openMVG_image
   openMVG_features
   openMVG_matching
   openMVG_system
   openMVG_multiview
   vlsift
-  stlplus)
+  ${POPSIFT_LIBRARIES}
```

Comment:
I've tried without the PUBLIC keywords and added cmake_policy(set CMP0028 New), and PopSift::popsift works just fine for me. Without any further fixes in the OpenMVG cmake